vignettes/meta_forestly.Rmd

```diff
@@ -0,0 +1,145 @@
+---
+title: "A closer look at meta_forestly()"
+output:
+  rmarkdown::html_document:
+    self_contained: no
+    number_sections: yes
+    code_folding: hide
+vignette: |
+  %\VignetteIndexEntry{Controlling Layout}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  message = FALSE,
+  warning = FALSE,
+  eval = TRUE,
+  echo = TRUE
+)
+```
+
+```{r}
+library(forestly)
+library(metalite)
+```
+
+# Overview
+
+`meta_forestly()` is the first step in the Interactive Forest Plot generation workflow and is a wrapper of several [metalite](https://merck.github.io/metalite/) functions. Specifically, it defines a metadata, including adam datesets, population, observations and parameters to be used in the forest plot. 
```

Comment:
- Why is "Interactive Forest Plot" in title case? If it's not a title, use regular case.
- Remove the link to pkgdown site.
- "metadata" is not a countable noun.
- "adam" should be stylized as "ADaM".

Reply:
why not to keep the link to metalite pkgdown site?

Reply:
It's ok to keep it.